deploy/provision.sh

```
@@ -0,0 +1,472 @@
+DAEMON_EOF
+systemctl restart docker
+
+echo "--- Droplet setup complete ---"
+REMOTE_SETUP
+
+# ---------------------------------------------------------------------------
+# Step 8: Create the spotify_mcp database
+# ---------------------------------------------------------------------------
+log "Step 8: Creating database '$DB_NAME'"
+
+ssh -o StrictHostKeyChecking=no "root@$DROPLET_IP" bash -s <<REMOTE_DB
```

Shellcheck (0.11.0)

[warning] Quote 'REMOTE_DB' to make here document expansions happen on the server side rather than on the client.

(SC2087)